app/models/adm.rb

```diff
@@ -1,7 +1,7 @@
-require "singleton"
+#require "singleton"
 
 class Adm < Person
-  include Singleton
+  #include Singleton
```

Comment:
why did you remove singleton?

Reply:
was commented so that the class can be instantiated normally, I will delete